README.md

```diff
@@ -8,144 +7,112 @@
+- [Officially Supported By](#officially-supported-by)
+- [Contributing](#contributing)
+- [License](#license)
 
-## Learning more
+## Status
 
-There are a few good resources to learn about Nu.
-There is a [book](https://www.nushell.sh/book/) about Nu that is currently in progress.
-The book focuses on using Nu and its core concepts.
+This project has reached a minimum-viable-product level of quality. Many people use it as their daily driver, but it may be unstable for some commands. Nu's design is subject to change as it matures.
 
-If you're a developer who would like to contribute to Nu, we're also working on a [book for developers](https://www.nushell.sh/contributor-book/) to help you get started.
-There are also [good first issues](https://github.com/nushell/nushell/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) to help you dive in.
+## Learning About Nu
 
-We also have an active [Discord](https://discord.gg/NtAbbGn) and [Twitter](https://twitter.com/nu_shell) if you'd like to come and chat with us.
+The [Nushell book](https://www.nushell.sh/book/) is the primary source of Nushell documentation. You can find [a full list of Nu commands in the book](https://www.nushell.sh/book/command_reference.html), and we have many examples of using Nu in our [cookbook](https://www.nushell.sh/cookbook/).
 
-You can also find information on more specific topics in our [cookbook](https://www.nushell.sh/cookbook/).
+We're also active on [Discord](https://discord.gg/NtAbbGn) and [Twitter](https://twitter.com/nu_shell); come and chat with us!
 
 ## Installation
 
-### Local
-
-Up-to-date installation instructions can be found in the [installation chapter of the book](https://www.nushell.sh/book/installation.html). **Windows users**: please note that Nu works on Windows 10 and does not currently have Windows 7/8.1 support.
-
-To build Nu, you will need to use the **latest stable (1.60 or later)** version of the compiler.
-
-Required dependencies:
-
--   pkg-config and libssl (only needed on Linux)
-    -   On Debian/Ubuntu: `apt install pkg-config libssl-dev`
-
-Optional dependencies:
-
--   To use Nu with all possible optional features enabled, you'll also need the following:
-    -   On Linux (on Debian/Ubuntu): `apt install libxcb-composite0-dev libx11-dev`
-
-To install Nu via cargo (make sure you have installed [rustup](https://rustup.rs/) and the latest stable compiler via `rustup install stable`):
-
-For Windows users, you may also need to install the [Microsoft Visual C++ 2015 Redistributables](https://docs.microsoft.com/cpp/windows/latest-supported-vc-redist).
-
-```shell
-cargo install nu
-```
-
-To install Nu via the [Windows Package Manager](https://aka.ms/winget-cli):
-
-```shell
-winget install nushell
-```
-
-To install Nu via the [Chocolatey](https://chocolatey.org) package manager:
+To quickly install Nu:
 
-```shell
-choco install nushell
+```bash
+# Linux and macOS
+brew install nushell
+# Windows
+winget install nu
```

Comment:
This conflicts what the documentation for winget https://www.nushell.sh/book/installation.html#package-managers

I'm not sure which is right.

Reply:
`Nushell.Nushell` is the unambiguous ID for Nushell on winget, probably easiest to use that. Fixed.
<img width="489" alt="image" src="https://user-images.githubusercontent.com/26268125/172462965-4f41c71f-eb46-4350-88f4-a716ed059893.png">

Reply:
I'll update the website in a bit.

Reply:
On second thought.... `winget install nushell` works and it's easier to memorize. Will go with that.